sqlmesh/core/engine_adapter/athena.py

```diff
@@ -297,15 +301,12 @@ def _truncate_table(self, table_name: TableName) -> None:
         self.execute(f"DELETE FROM {table.sql(dialect=self.dialect, identify=True)}")
 
     def _table_type(
-        self, table_properties: t.Optional[t.Dict[str, exp.Expression]] = None
+        self, table_type: t.Optional[str] = None
     ) -> t.Union[t.Literal["hive"], t.Literal["iceberg"]]:
-        """
-        Use the user-specified table_properties to figure out of this is a Hive or an Iceberg table
-        """
+        if table_type and table_type.lower() == "iceberg":
+            return "iceberg"
+
         # if we cant detect any indication of Iceberg, this is a Hive table
-        if table_properties and (table_type := table_properties.get("table_type", None)):
```

Comment:
This became a lot simpler, we now dont need to try and parse the table type out of a `physical_properties` entry